packages/genesys-spark-components/src/components/stable/gux-button/gux-button.scss

```diff
@@ -103,6 +103,14 @@ button {
     border-style: var(--gse-ui-button-tertiary-default-border-style);
     border-width: var(--gse-ui-button-tertiary-default-border-width);
 
+    &.gux-icon-only {
+      padding: calc(
+        var(--gse-ui-button-paddingIconOnly) - var(
+            --gse-ui-button-tertiary-default-border-width
+          )
+      );
```

Comment:
99% of the time linters are great but 1% of the time the can really produce some ugly formatting :).